src/actions/articlesActions.js

```diff
@@ -21,46 +26,92 @@ const postArticleSuccess = response => ({
 
 const fetchAnArticleFailure = err => ({
   type: FETCH_AN_ARTICLE_FAILURE,
-  payload: err.response.data.errors,
+  payload: 'Errors have occured',
 });
 
 const fetchAnArticleSuccess = response => ({
   type: FETCH_AN_ARTICLE_SUCCESS,
   payload: response.data,
 });
 
-const baseURL = 'https://ah-backend.herokuapp.com/api';
+const deleteArticleSuccess = response => (
+  {
+    type: DELETE_AN_ARTICLE_SUCCESS,
+    payload: 'successfully deleted an article',
+  }
+);
 
+const deleteArticleFailure = err => (
+  {
+    type: DELETE_AN_ARTICLE_FAILURE,
+    payload: 'Errors have occured',
+  }
+);
 
-export const postArticle = (articleData, history) => dispatch => (
-  axiosInstance.post('articles/', articleData)
-    .then((response) => {
-      dispatch(postArticleSuccess(response));
-      notify.show('Article created Successfully', 'success', 4000);
-      history.push('/dashboard');
-    }).catch((err) => {
-      dispatch(postArticleFailure(err.response.data));
-      notify.show('Errors have occured', 'error', 4000);
-    })
+const editArticleSuccess = response => (
+  {
+    type: EDIT_AN_ARTICLE_SUCCESS,
+    payload: response.data,
+  }
 );
 
-export const fetchArticles = () => dispatch => (
-  axios.get(`${baseURL}/articles/`)
-    .then((response) => {
-      dispatch({
-        type: GET_ARTICLES,
-        payload: response.data.results,
-      });
-    })
+const editArticleFailure = err => (
+  {
+    type: EDIT_AN_ARTICLE_FAILURE,
+    payload: 'Errors have occured',
+  }
 );
-export const getSingleArticle = slug => (dispatch) => {
-  axios.get(`${baseURL}/articles/${slug}`)
-    .then((response) => {
-      dispatch(fetchAnArticleSuccess(response));
-    }).catch((err) => {
-      dispatch(fetchAnArticleFailure(err.response.data));
-      notify.show('Errors have occured', 'error', 4000);
+
+
+const baseURL = 'https://ah-backend.herokuapp.com/api';
```

Comment:
@felixkiryowa kindly use the axios intance in src

Reply:
This baseURL is being used for those endpoints that do not need user authorization like fetching articles and getting a single single